qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -597,19 +597,18 @@ def _common_creation_steps(cls, md_template, obj_id):
 
             # Create table with custom columns
```

Comment:
Comment should be changed to, "Create table and populate with metadata key/value pairs" or similar.